src/System.Private.Reflection.Core/src/System/Reflection/Runtime/General/MetadataReaderExtensions.NativeFormat.cs

```diff
@@ -326,49 +326,61 @@ public static Object ParseConstantValue(this Handle handle, MetadataReader reade
             return value;
         }
 
+        public static Object ParseConstantNumericValue(this Handle handle, MetadataReader reader)
+        {
+            HandleType handleType = handle.HandleType;
```

Comment:
Nit: It can  be a direct `switch (handle.HandleType)`, not need to cache it in a local.

I know few other existing places in the file do it as well; not all of them though.